app/src/main/java/net/squanchy/speaker/domain/view/Speaker.java

```diff
@@ -3,24 +3,21 @@
 import com.google.auto.value.AutoValue;
 
 import net.squanchy.service.firebase.model.FirebaseSpeaker;
+import net.squanchy.support.lang.Ids;
+import net.squanchy.support.lang.Optional;
 
 @AutoValue
 public abstract class Speaker {
 
-    public static Speaker create(FirebaseSpeaker speaker) {
-        return new AutoValue_Speaker(speaker.speakerId, speaker.firstName,
-                speaker.lastName, speaker.avatarImageURL);
+    public static Speaker create(FirebaseSpeaker speaker, long numericSpeakerId) {
+        return new AutoValue_Speaker(numericSpeakerId, speaker.id, speaker.name, Optional.fromNullable(speaker.photo_url));
     }
 
-    public abstract long id();
+    public abstract long numericId();
```

Comment:
AAAAAH It's for the `SpeakersActivity` (which is going away), but will come in handy for the search I suppose. 👍